backend-go/api/codepair/v1/models/errors.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import "errors"
+
+func RequiredFieldError(field string) error {
+	return errors.New(field + " is required")
+}
+
+func MinLengthError(field string, min int) error {
+	return errors.New(field + " must be at least " + string(min) + " characters")
```

Comment:
You should use `strconv` to convert an integer to a string.

```go
string(97) // rune 'a'
strconv.Itoa(97) // string "97"
```

Alternatively, you can use `fmt.Sprintf` to create a new string

```go
fmt.Sprintf("%s must be at least %d characters", field, min)
```